lib/asciidoctor/syntax_highlighter.rb

```diff
@@ -0,0 +1,80 @@
+module Asciidoctor
+  module SyntaxHighlighter
+    PROVIDED = RUBY_ENGINE_OPAL ? {} : {
```

Comment:
So we have to provide/implement a syntax highlighter in Asciidoctor.js (ie. Highlight.js) ?

Reply:
You just need to require one of them in the Asciidoctor.js build (since Opal can't require dynamically). But I think I'm going to require highlight.js statically since that's the most common choice and it works across all the platforms. So look for that change.

Reply:
Oh right, I'm silly :grimacing: 

>  But I think I'm going to require highlight.js statically since that's the most common choice and it works across all the platforms. So look for that change.

:+1:

Reply:
:+1: